app/helpers/writable_helper.rb

```diff
@@ -55,6 +55,22 @@ def privacy_icon(privacy)
     image_tag("icons/#{img}.png", class: 'vmid', title: name)
   end
 
+  STATUS_MAP = {
+    Post::STATUS_ACTIVE    => ['In Progress', 'book_open'],
+    Post::STATUS_COMPLETE  => ['Complete', 'book'],
+    Post::STATUS_HIATUS    => ['On Hiatus', 'hourglass'],
+    Post::STATUS_ABANDONED => ['Abandoned', 'book_grey'],
+  }
+
+  def status_state(status, only_icon: false, only_text: false, short_title: false)
+    name = STATUS_MAP[status][0]
+    img = STATUS_MAP[status][1]
```

Comment:
Maybe we should do some kind of check to make sure `status` can be found in `STATUS_MAP`, or else error, so we can catch mistakes better in development? I guess we'd get it anyway, as a NilMethodError (`STATUS_MAP[status][0]` would be `nil[0]`), but probably better to be explicitly.

Reply:
What should I have it do if it can't find it?

Reply:
We can raise an error, since it should be caught in development or else we should see error messages loudly when we push a broken change.